Makefile.gnu: allow non-root install by not enforcing root ownserhip

Signed-off-by: Rémi Rérolle <remi.rerolle@gmail.com>

diff -ruN a/Makefile.gnu b/Makefile.gnu
--- a/Makefile.gnu	2015-04-09 16:34:02.315316841 +0200
+++ b/Makefile.gnu	2015-04-09 16:34:28.875483201 +0200
@@ -71,9 +71,9 @@
 
 install:
 	install -d $(INCDIR) $(INSTALLDIR)
-	install -m 644 -o root -g root $(HEADER) $(INCDIR)
-	install -m 644 -o root -g root $(STATICLIB) $(INSTALLDIR)
-	install -m 755 -o root -g root $(SHAREDLIB) $(INSTALLDIR)
+	install -m 644 $(HEADER) $(INCDIR)
+	install -m 644 $(STATICLIB) $(INSTALLDIR)
+	install -m 755 $(SHAREDLIB) $(INSTALLDIR)
 	ln -sf $(SHAREDLIB) $(INSTALLDIR)/$(VERLIBNAME)
 	ln -sf $(VERLIBNAME) $(INSTALLDIR)/$(LIBNAME)	
 #	ldconfig
